package.json

```diff
@@ -3,16 +3,17 @@
   "version": "1.0.0",
   "description": "Tell Slack you're working from home today 🏡",
   "repository": {
-	  "type": "git",
-	  "url": "https://github.com/Symphono/wfh-slack-command"
+    "type": "git",
+    "url": "https://github.com/Symphono/wfh-slack-command"
   },
   "engine": {
     "node": "7.6.0"
   },
   "scripts": {
     "start": "node src/index.js",
     "test-watch": "mocha --debug-brk --inspect --watch",
-	"test": "mocha"
+    "test-debug": "mocha --inspect-brk",
```

Comment:
well - it depends.  i think the ability for watch is workflow dependent.  when you do your debugging, do you keep the process running with watch?  I guess we can just get rid of `test-debug`, which is a one time debugging session.